custom/icds_reports/static/js/directives/sortable-kpi/sortable-kpi.directive.js

```diff
@@ -0,0 +1,27 @@
+var url = hqImport('hqwebapp/js/initial_page_data').reverse;
+
+
+function SortableKpiController($rootScope, $scope) {
+    this.expandedCard = -1;
```

Comment:
nit: might be more readable if you had `const = NO_CARD_EXPANDED = -1`. or alternatively, why use `-1` and not `null` or `undefined`?

Reply:
Yeah It can be null or undefined. I should have used strict comparision "===" rather than "==". where null and 0 are considered same.

I will make the change